### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by tibi77**
### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by tibi77** [comment]
Reparteeing the links that were not work on the readme

File: README.rst [hunk 1]
```diff
@@ -19,12 +19,27 @@ algorithms. To learn more about the different languages supported and the
...
+
+Twitter_
+
+Demo_
+
 To see what coala can do for your language, run:
 
+
+
+
```
Comment (sils) [inline README.rst]: why three newlines?

File: README.rst [hunk 2]
```diff
@@ -19,12 +19,27 @@ algorithms. To learn more about the different languages supported and the
...
 
+Website_
+
+Facebook_
+
+Documentation_
+
+Twitter_
+
+Demo_
```
Comment (sils) [inline README.rst]: can't you rather fix the table below instead of leaving it broken?

**Comment by Mixih** [comment]
You have two pulls with the same please close one...

**Comment by yash-nisar** [comment]
High inactivity on this one. @Adrianzatreanu

**Comment by yash-nisar** [comment]
@meetmangukiya @aptrishu High inactivity on this PR.

**Comment by aptrishu** [comment]
I don't know the issue related to this PR (if there is any).
So, if there was no issue then leave it as it is..

**Pull request closed without merging**